temperature_regulator/project/main/server_configuration.config

```diff
@@ -0,0 +1,8 @@
+listener socket address : 127.0.0.1
+listener socket port : 8080
+listener socket timeout : 1  # For unblocking
+max connections : 10
+
+text encoding : UTF-8
```

Comment:
jakby co to ja nie używam żadnego encodingu więc z komuniacja ze mna (urzadzeniem temperatury) może być problem jezeli bedziesz cos kodował w UTF-8